frontend/mulighetsrommet-veileder-flate/src/App.module.scss

```diff
@@ -1,22 +1,37 @@
-.app_container {
+.amt_container {
```

Comment:
`amt` er namespace til komet. Det er jo også en grei forkortelse for arbeidsmarkedstiltak. Usikker på om det er nyttig eller uheldig om vi begynner å ta i bruk denne konvensjonen (vi har jo også brukt `mr` som forkortelse for mulighetsrommet, som all koden tross alt ligger under fortsatt) :thinking:

Reply:
Ja, tenkte siden vi snakka om å kalle den for arbeidsmarkedstiltak så var det en grei forkortelse. Skal jeg beholde eller bytte til mr?